project-resources/PROJECT_FUNDS.md

```diff
@@ -0,0 +1,24 @@
+# OpenJS Foundation Project-specific Initiative Funds
+
+The OpenJS Foundation provides a set of monetary funds for projects of the OpenJS Foundation. 
+These funds are designed to support collaboration and incentive project-specific initiatives that have high-impact within the individual project's scope.
+
+Alongside these lines, the fund exist to support the development of these 
+initiatives and give visibility/recognition to the champions of these project-specific initiatives, highligthing the individuals behind them.
+
+## F.A.Q.
+
+#### How might a project request for funds?
+
+Funds should be requested on an ad-hoc basis through a PR requesting the funds. The requests do not need to follow a specific template,
+but the description should contain relevant information regarding the project initiative, 
+and any information that might be useful to highlight the need of funds and their benefits.
+
+#### How these requests get evaluated?
+
+The requests should go through the OpenJS Foundation CPC (Cross Project Council) and must have approval from the CPC members. 
+Finally the request should be forwarded to the Board of Directors for final approval.
+
+#### How much might a project request?
+
+Dollar amounts will be determined on ad-hoc basis (proposed cap isn't necessary). Requests should be reasonable and can be discussed.
```

Comment:
It would be interesting to have a range of what is feasible to be asked. And if different project categories (At-large, Impact, Incubation) should have different limits.

Reply:
> And if different project categories (At-large, Impact, Incubation) should have different limits.

Would a note clarifying that project status will affect the allocated budget help?

Reply:
> Would a note clarifying that project status will affect the allocated budget help?

I think that's not necessarily true and also not super useful.

Reply:
I don't think we have a good answer for this right now. :(